examples/hybrid/box/bubble_3d_invariant.jl

```diff
@@ -321,7 +321,7 @@ function linkfig(figpath, alt = "")
     end
 end
 
-dir = "bubble3d_invariant"
+dir = "bubble_3d_invariant"
```

Comment:
```suggestion
dir = "bubble_3d_invariant_potential_temperature"
```

Reply:
can you also change the name of the file adding 'potential_temperature' to be consistent with the total energy version.